lib/packages.dart

```diff
@@ -34,34 +32,6 @@ abstract class Packages {
   factory Packages(Map<String, Uri> packageMapping) =>
       new MapPackages(packageMapping);
 
```

Comment:
How about removing this constructor as well, and just not import any implementations. I'd be fine with that.